packages/libs/react-components/src/components/Input/InputGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import { AlertBox } from '../Icons/svgs/AlertBox';
+import { AlertCircleOutline } from '../Icons/svgs/AlertCircleOutline';
+
+import {
+  StyledHelper,
+  StyledInfo,
+  StyledInputGroupHeader,
+  StyledInputGroupWrapper,
+  StyledLabel,
+  StyledTag,
+} from './styles';
+
+import classnames from 'classnames';
+import React, { FC } from 'react';
+
+export interface IInputGroupProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, 'as'> {
+  label?: string;
+  tag?: string;
+  info?: string;
+  helper?: string;
+  status?: 'success' | 'error';
+  disabled?: boolean;
+  children: React.ReactNode;
+}
+
+export const InputGroup: FC<IInputGroupProps> = ({
+  label,
+  tag,
+  info,
+  helper,
+  children,
+  status,
+  disabled,
+  className,
+  ...rest
+}) => {
+  return (
+    <StyledInputGroupWrapper
+      {...rest}
+      className={classnames({ disabled }, className)}
```

Comment:
I believe the functionality you are achieving by using className can alternatively be achieved by usage of stitches variants which is what we are doing in other components
https://stitches.dev/docs/variants

I think using this feature is preferable to adding the classnames package for the sake of consistency and having fewer dependencies

Reply:
yes, but the issue is it will regenerate styles every time, therefore in a page with many components, running it on browser causes lag, but with classnames you'll have style there, and it'll just changes it's look, so there is no style regeneration

Reply:
Yeah, I understand your rationale and I think it's a good approach, but it doesn't align with the standard we already have. In this circumstance where we will not develop this package further, I would prefer to keep things consistent

I think this might be a good approach for the next component library though!